mantle/cli/cli.go

```diff
@@ -102,6 +102,10 @@ func WrapPreRun(root *cobra.Command, f PreRunEFunc) {
 		if err := f(cmd, args); err != nil {
 			return err
 		}
+		// Always inject startLogging to commands that are wrapping the preRun
+		// due to github.com/spf13/cobra/issues/253 where parent command's
+		// preRun & preRunE functions are overwritten by children
```

Comment:
should we link to https://github.com/coreos/coreos-assembler/issues/1321 here too?

Reply:
Wouldn't hurt, but it'll also be there in the git annotate.